src/api/views.py

```diff
@@ -89,3 +89,26 @@ def available(request, domain=""):
             return JsonResponse({"available": False, "message": DOMAIN_API_MESSAGES["unavailable"]})
     except Exception:
         return JsonResponse({"available": False, "message": DOMAIN_API_MESSAGES["error"]})
+
+
+@require_http_methods(["GET"])
+@login_not_required
+def get_current_full(request, file_path="migrationdata/current-full.csv"):
```

Comment:
Add comment about what this file should contain, a dev looking at this a year from now would have to do some detective work to understand the purpose and what this file is